cli/src/config/templates.toml

```diff
@@ -41,10 +41,15 @@ tag_list = '''
 label("tag", name) ++ format_ref_targets(self) ++ "\n"
 '''
 
+# TODO: "op_log" is not a great label name for op_summary. We could replace it
+# with `operation` and insert this new label throughout the op log templates.
 op_summary = '''
-if(root,
-  separate(" ", self.id().short(), label("root", "root()")),
-  separate(" ", self.id().short(), format_time_range(self.time()), self.description().first_line())
+label(
+  "op_log",
+  if(root,
+    separate(" ", self.id().short(), label("root", "root()")),
+    separate(" ", self.id().short(), format_time_range(self.time()), self.description().first_line())
+  )
```

Comment:
nit: This can be added to `WorkspaceCommandHelper::operation_summary_template()` instead.
```rust
self.reparse_valid_template(...).labeled("op_log")
```

Reply:
Sure.

In the longer term, I think it'd be nice to move all of these into `templates.toml` so that all the labels are in one place. Right now, I wouldn't know where to look (the debug output, I suppose).

Reply:
It seems that `op diff` already inserted the `op_log` template. I'll make it so that it only gets inserted once, but this reinforces my desire to put all the labels in one place.

Reply:
> In the longer term, I think it'd be nice to move all of these into templates.toml

The idea is that templates can be customized by user, but the outermost label shouldn't overridden by such changes.

Reply:
I see. Maybe it's possible to consistently move all the labels to functions like `operation_summary_template` where they'd all be in the same place. Or, we could have `_operation_summary_template_internal` template that wraps `operation_summary_template` in the label.

Reply:
I would avoid adding abstraction layers in the template configuration. It's easier to do in code so long as the requirement isn't dynamic.